dojo/tools/jfrog_xray_api_summary_artifact/parser.py

```diff
@@ -0,0 +1,170 @@
+import json
+import re
+
+from cvss import CVSS3
+
+from dojo.models import Finding
+
+
+class JFrogXrayApiSummaryArtifactParser(object):
+
+    # This function return a list of all the scan_type supported by your parser. This identifiers are used internally. Your parser can support more than one scan_type
+    # For example some parsers use different identifier to modify the behavior of the parser (aggregate, filter, etc…)
+    def get_scan_types(self):
+        return ["JFrog Xray API Summary Artifact Scan"]
+
+    # This function return a string used to provide some text in the UI (short label)
+    def get_label_for_scan_types(self, scan_type):
+        return scan_type
+
+    # This function return a string used to provide some text in the UI (long description)
+    def get_description_for_scan_types(self, scan_type):
+        return "Import Xray findings in JSON format from the JFrog Xray API Summary/Artifact JSON response"
+
+    # This function return a list of findings
+    def get_findings(self, json_output, test):
+        tree = json.load(json_output)
+        return self.get_items(tree, test)
+
+    def get_items(self, tree, test):
+        items = []
+        if 'artifacts' in tree:
+            artifact_tree = tree['artifacts']
+            for artifactNode in artifact_tree:
+                artifact_general = artifactNode['general']
+                artifact_issues = artifactNode['issues']
+                for node in artifact_issues:
+                    service = decode_service(artifact_general['name'])
+                    item = get_item(node, str(service), test)
+                    items.append(item)
+
+        return items
+
+
+# Retrieve the findings
+def get_item(vulnerability, service, test):
+    cve = "No CVE"
+    cwe = None
+    cvssv3 = None
+    cvss_v3 = "No CVSS v3 score"
+    cvssv3_score = 0.0
+    unique_id_from_tool = None
+    impact_paths = None
+    impact_path = ImpactPath("", "", "")
+
+    if 'severity' in vulnerability:
+        if vulnerability['severity'] == 'Unknown':
+            severity = "Informational"
+        else:
+            severity = vulnerability['severity'].title()
+    else:
+        severity = "Informational"
+
+    # Some entries have no CVE entries, despite they exist. Example CVE-2017-1000502.
+    cves = vulnerability.get('cves', [])
```

Comment:
you can now set multiple identifiers in the vulnerability references relation

Reply:
Hi, not sure what you mean by this comment.

Reply:
You can add all CVEs from the report to the `vulnerability_reference_set` field of a Finding, check https://github.com/DefectDojo/django-DefectDojo/pull/5904

Reply:
Having deployment issues at this point so I can't test this on my machine. Added a question on the [defectdojo slack channel](https://owasp.slack.com/archives/C2P5BA8MN/p1650980167639499). Could we complete this PR and I will create another one once I'm up and running again?